test/server-unit/pdf.spec.js

```diff
@@ -3,7 +3,7 @@ const { expect } = require('chai');
 const rewire = require('rewire');
 const util = require('util');
 const path = require('path');
-const fs = require('fs');
+const fs = require('mz/fs');
```

Comment:
What is the difference between `fs` and `mz/fs`

Reply:
[mz/fs](https://www.npmjs.com/package/mz) uses promises, so that I can use it with async/await like you are using here.